src/main/java/chess/view/OutputView.java

```diff
@@ -0,0 +1,51 @@
+package chess.view;
+
+import chess.domain.dto.PieceResponse;
+import chess.domain.piece.Color;
+import chess.domain.piece.Type;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class OutputView {
+    private static final char EMPTY_SQUARE = '.';
+    private static final int BOARD_SIZE = 8;
+
+    public void printGameStartMessage() {
+        System.out.println("체스 게임을 시작합니다.");
+    }
+
+    public void printBoard(final List<PieceResponse> pieces) {
+        char[][] board = setUpBoard();
+        addPieceToBoard(pieces, board);
+        printBoardStatus(board);
+        System.out.println();
+    }
+
+    private char[][] setUpBoard() {
+        char[][] board = new char[BOARD_SIZE][BOARD_SIZE];
+        for (char[] line : board) {
+            Arrays.fill(line, EMPTY_SQUARE);
+        }
+        return board;
+    }
+
+    private void addPieceToBoard(final List<PieceResponse> pieces, final char[][] board) {
+        for (PieceResponse response : pieces) {
+            int y = response.getRankIndex();
+            int x = response.getFileIndex();
+            board[y][x] = getPieceDisplay(response.getType(), response.getColor());
+        }
+    }
+
+    private char getPieceDisplay(final Type type, final Color color) {
+        return PieceView.findByType(type).changeToView(color);
+    }
+
+    private void printBoardStatus(final char[][] board) {
+        IntStream.range(0, board.length)
+                .mapToObj(lineCount -> board[board.length - 1 - lineCount])
+                .map(String::new)
+                .forEach(System.out::println);
+    }
```

Comment:
사소한 내용이긴한데 Status는 빼도 될 것 같습니다.
개인적으로 Status는 진행중, 진행완료, 준비 이런 보드의 상태가 떠올라요.

```suggestion
    private void printBoard(final char[][] board) {
        IntStream.range(0, board.length)
                .mapToObj(lineCount -> board[board.length - 1 - lineCount])
                .map(String::new)
                .forEach(System.out::println);
    }
```

Reply:
말씀대로 Status가 붙으면서 더 어색해지는 느낌이 드는 것 같아 제거했습니다
사소한 부분까지 꼼꼼한 리뷰 감사드립니다 😄